util/data_stream.go

```diff
@@ -42,6 +42,9 @@ type DataStream struct {
 	Title   string `config:"title" json:"title" validate:"required"`
 	Release string `config:"release" json:"release"`
 
+	Description string  `config:"description" json:"description"`
```

Comment:
In general `BasePackage` and `DataStream` correspond to their format in package-spec. I wouldn't extend them this way. If you need to add more fields to view, but not manifest, I would consider creating new structures (possibly extending ones in spec).